src/main.cpp

```diff
@@ -632,7 +632,6 @@ int main ( int argc, char** argv )
     QCoreApplication* pApp = new QCoreApplication ( argc, argv );
 #else
 # if defined ( Q_OS_IOS )
-    bIsClient = false;
     bUseGUI = true;
```

Comment:
Is this needed?

Reply:
no, and in the last version of the PR, this line has been removed.

Reply:
I can still see bUseGui = true in the diff?

Reply:
I can image bUseGui is forced to true, independent of command line settings. Apps on IOS must have a GUI.